Formula/php.rb

```diff
@@ -0,0 +1,333 @@
+      --with-gd
+      --with-gettext=#{Formula["gettext"].opt_prefix}
+      --with-fpm-user=_www
+      --with-fpm-group=_www
+      --with-iconv-dir=/usr
+      --with-icu-dir=#{Formula["icu4c"].opt_prefix}
+      --with-jpeg-dir=#{Formula["jpeg"].opt_prefix}
+      --with-kerberos=/usr
+      --with-ldap=shared
+      --with-ldap-sasl=/usr
+      --with-libxml-dir=/usr
+      --with-imap=shared,#{Formula["imap-uw"].opt_prefix}
+      --with-imap-ssl=#{Formula["openssl"].opt_prefix}
+      --with-libzip=#{Formula["libzip"].opt_prefix}
+      --with-mhash
+      --with-mysql-sock=/tmp/mysql.sock
+      --with-mysqli=mysqlnd
+      --with-pdo-mysql=mysqlnd
+      --with-pdo-odbc=unixODBC,#{Formula["unixodbc"].opt_prefix}
+      --with-ndbm=/usr
+      --with-openssl=#{Formula["openssl"].opt_prefix}
+      --with-pdo-dblib=#{Formula["freetds"].opt_prefix}
+      --with-pdo-pgsql=#{Formula["libpq"].opt_prefix}
+      --with-pgsql=#{Formula["libpq"].opt_prefix}
+      --with-png-dir=#{Formula["libpng"].opt_prefix}
+      --with-pspell=#{Formula["aspell"].opt_prefix}
+      --with-snmp
+      --with-tidy=shared,#{Formula["tidy-html5"].opt_prefix}
+      --with-unixODBC=#{Formula["unixodbc"].opt_prefix}
+      --with-webp-dir=#{Formula["webp"].opt_prefix}
+      --with-pic
+      --with-xmlrpc
+      --with-zlib=/usr
+      --with-libedit
+      --with-xsl=/usr
+    ]
+
+    if MacOS.version < :lion
+      args << "--with-curl=#{Formula["curl"].opt_prefix}"
+    else
+      args << "--with-curl"
+    end
+
+    args << "--enable-maintainer-zts" if build.with? "thread-safety"
+
+    if build.devel?
+      args += %W[
+        --with-password-argon2=#{Formula["argon2"].opt_prefix}
+        --with-sodium=#{Formula["libsodium"].opt_prefix}
+      ]
+    else
+      args << "--with-mcrypt=shared,#{Formula["mcrypt"].opt_prefix}"
+    end
+
+    system "./configure", *args
+
+    inreplace "Makefile" do |s|
+      # Custom location for php module and remove -a (don't touch httpd.conf)
+      s.gsub! /^INSTALL_IT = \$\(mkinstalldirs\) '([^']+)' (.+) LIBEXECDIR=([^\s]+) (.+) -a (.+)$/,
+        "INSTALL_IT = $(mkinstalldirs) '#{lib}/httpd/modules' \\2 LIBEXECDIR='#{lib}/httpd/modules' \\4 \\5"
+
+      # Reorder linker flags to put system paths at the end to avoid accidential system linkage
+      %w[EXTRA_LDFLAGS EXTRA_LDFLAGS_PROGRAM].each do |mk_var|
+        system_libs = []
+        other_flags = []
+        s.get_make_var(mk_var).split.each do |f|
+          if f[%r{^-L\/(?:Applications|usr\/lib)\/}]
+            system_libs << f
+            next
+          end
+          other_flags << f
+        end
+        s.change_make_var! mk_var, other_flags.concat(system_libs).join(" ")
+      end
+    end
+
+    # Shared module linker flags come after the sytem flags, prepend to avoid accidential system linkage
+    ENV.prepend "LDFLAGS", "-L#{Formula["tidy-html5"].opt_lib}"
+
+    system "make"
+    ENV.deparallelize
```

Comment:
This should be reported upstream with a link to the report.

Reply:
I think we should try running it without deparallize in that case and see if it breaks. I think the configuration has been there for some time and as far as I could see there's no proper documentation of the breakage so I'd opt to remove it and hope for the best =) If not then lets report it upstream and get it properly fixed.

Reply:
I'm pretty confident parallel install is still broken. I'll run through the build to capture the logs, but if I recall it may have something to do with the timing of the pear download and cli binary.